src/main/java/com/softserve/itacademy/controller/CourseController.java

```diff
@@ -0,0 +1,40 @@
+package com.softserve.itacademy.controller;
+
+import com.softserve.itacademy.dto.CourseDto;
+import com.softserve.itacademy.service.CourseService;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+import static org.springframework.http.HttpStatus.OK;
+
+@RestController
+@RequestMapping("/api/courses")
```

Comment:
You didn't read carefully the resources I sent in chat. How about versioning of the API?